variants/nina_w10/pins_arduino.h

```diff
@@ -23,10 +23,10 @@ static const uint8_t RX = 3;
 static const uint8_t SDA = 12;
 static const uint8_t SCL = 13;
 
-static const uint8_t SS    = 5;
-static const uint8_t MOSI  = 23;
-static const uint8_t MISO  = 19;
-static const uint8_t SCK   = 18;
+static const uint8_t SS    = 34;
```

Comment:
shouldn't this change be being made in variants/nora_w10/pins_arduino.h that was added via https://github.com/espressif/arduino-esp32/pull/7191/files?

Reply:
Thanks for letting me know, my mistake, here is the correct fix. 
#7208